python-template/{{cookiecutter.placeholder_repo_name}}/.github/workflows/upgrade-python-requirements.yml

```
@@ -0,0 +1,24 @@
+name: Upgrade Requirements
+
+on:
+  schedule:
+     - cron: "0 2 * * 1"
+  workflow_dispatch:
+     inputs:
+       branch:
+         description: 'Target branch to create requirements PR against'
+         required: true
+         default: 'master'
```

```suggestion
         default: 'main'
```
I believe cookiecutters are now generating repos with a default branch of `main`.